lib/prepare_business_uk_leaving_eu.yaml

```diff
@@ -0,0 +1,129 @@
+---
```

Comment:
Should this file be in this PR?  If we're not going live yet, it seems we want this feature but without this finder.

Reply:
Yup, the Q&A won't be live yet though since that will involve publishing the route from rummager.